tests/test_directives.py

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2017 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+import pytest
+import sys
+from copy import deepcopy
+
+from flow.directives import Directives
+from flow.directives import NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS
+
+
+@pytest.fixture()
+def setUp(test_class=Directives, **kwargs):
+    def _setup(test_class=Directives, **kwargs):
+        return test_class(**kwargs)
+    return _setup
+
+
+available_directives_list = [NP, NRANKS, NGPU, EXECUTABLE, OMP_NUM_THREADS]
+
+
+class TestItems:
+    """
+    Tests for _DirectivesItem class
+    """
+    def test_serial(self):
+        assert NP.serial(4, 2) == 4
```

Comment:
I like this test. However, we need it for all all caps `_DirectiveItems`.

Reply:
Inserted tests for every directive

Reply:
I do not see any new tests here.